ui/js/directives/treeherder/clonejobs.js

```diff
@@ -912,11 +914,12 @@ treeherder.directive('thCloneJobs', [
                     platform.name,
                     platform.option
                 );
-                // We first determine whether the row has some visible element
+
                 var display_style = "none";
-                platform.groups.forEach(function(group) {
-                    group.jobs.forEach(function(job) {
-                        if (filterWithRunnable(job)) {
+                // We first determine whether the row has some visible element
+                platform.groups.forEach(function(jobGroup) {
+                    jobGroup.jobs.forEach(function(job) {
+                        if (job.visible) {
                             display_style = "table-row";
                         }
```

Comment:
Nothing to do with your patch, but this is a little inefficient since it has to iterate through every element. Lodash's "some" function would be much better (https://lodash.com/docs#some). Since we're here we might as well do something like this:

```js
// We first determine whether the row has some visible element
var anyVisible = _.some(platform.groups, function(jobGroup) {
  return _.some(jobGroup.jobs, {visible: true});
});
var display_style = anyVisible ? "table-row" : "none";
```